viewslides.py

```diff
@@ -1124,11 +1017,11 @@ def save_extracted_file(self, zipfile, filename):
         if (outfn == ''):
             return False
         fname = os.path.join(self.get_activity_root(), 'instance', outfn)
-        f = open(fname, 'w')
+        f = open(fname, 'wb')
         try:
             f.write(filebytes)
         finally:
-            f.close
+            f.close()
```

Comment:
Do you think it would be nice to use `with.. as` to write files, which could implicitly do many of these tasks?

```python

with open(fname, 'wb') as fp:
    fp.write(filebytes)
 ```

Reply:
Yeah I agree, thanks for pointing that out. Fixed in 6077947.